src/components/Profile/VpDelegatorsBox.tsx

```diff
@@ -30,7 +30,7 @@ export default function VpDelegatorsBox({
 }: Props) {
   const t = useFormatMessage()
   const [userAddress] = useAuthContext()
-  const isLoggedUserProfile = userAddress === address
+  const isLoggedUserProfile = userAddress?.toLowerCase() === address?.toLowerCase()
```

Comment:
this is tricky... if both can be null/undefined, it could return true.

You can use `isSameAddress(userAddress, address)` :)

Reply:
Thank you! I forgot about that